lib/shopify-cli/commands/authenticate.rb

```diff
@@ -22,10 +23,27 @@ def call(args, _name)
         end
       end
 
-      def run_task(task_to_run)
-        CLI::UI::Spinner.spin("Waiting to authenticate") do |spinner|
-          spinner.update_title("Authetication token stored")
-          task_to_run
+      def run_identity
+        opt = CLI::UI::Prompt.confirm('Reauthenticate with Partner Dashboard?')
+        if opt
+          CLI::UI::Spinner.spin("Waiting to Reauthenticate with Partner Dashboard") do |spinner|
+            spinner.update_title("Reauthenticated with Partner Dashboard")
+            ShopifyCli::Tasks::AuthenticateIdentity.call(@ctx)
+          end
+        else
+          abort
```

Comment:
These should be `raise ShopifyCli::AbortSilent` instead?

Reply:
sure should